### PR TITLE
June 2023 newsletter - open for ideas

### PR DESCRIPTION
**Pull request by benjaoming**
* [x] Add all roadmap items deployed
* [x] Security advisories
* [x] Upcoming changes
* [x] Awesome project of the month
* [x] Possible issues
* [x] TLS 1.2 update
* [x] /en/ to /en/latest/ redirect
* [x] Attractive screenshot

### CODE REVIEW

**Comment by humitos** [comment]
We should probably mention https://github.com/readthedocs/readthedocs.org/issues/10351 in the newsletter and point users to this blog post https://github.com/readthedocs/blog/pull/219 so they can add the `.readthedocs.yaml` file to their projects.

File: newsletter-june-2023.rst [hunk 1]
```diff
@@ -10,8 +10,14 @@ Read the Docs newsletter - June 2023
 News and updates
```
Comment (stsewd) [inline newsletter-june-2023.rst:10]: Maybe is worth mentioning that we have migrated all custom domains to have 1.2 as the minimum TLS version?

The `/en/`to `/en/latest/` redirect could also be interesting to mention

File: newsletter-june-2023.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+- 🚢️ We have concluded work on the first phase of our new `readthedocs-client <https://github.com/readthedocs/readthedocs-client>`_.
+  The client will be a single library for all the ways Read the Docs enhances documentation from having a flyout menu
+  to link to other versions, translations, or subprojects, to showing version warnings for folks browsing outdated docs
+  to running ads on community projects.
+  There will be integration points and APIs to customize all of these features in the new client.
+
+  We will start to use it on a few selected projects.
+  If you are interested in having it enabled on your project,
+  please `reach out`_.
```
Comment (benjaoming) [inline newsletter-june-2023.rst:37]: Trying again to stir a bit of hype around this :)

Can you suggest changes that remove any over-promising? And can we have another nice screenshot here? (this newsletter would like a screenshot)

CC: @humitos @agjohnson I promised to bring this up again after it was postponed last time :innocent:  :wave:

Comment (humitos) [reply newsletter-june-2023.rst:37]: This is not 100% accurate. The new client is enabled automatically for projects using `build.commands` now and it's not feature complete (no flyout, for example). However, we need to work around a CF quota limitation.

In other words, I think this is not ready to be announced here yet.

Comment (benjaoming) [reply newsletter-june-2023.rst:37]: It's great to have something in the upcoming horizon that makes readers know that we're doing cool things. I'll try to find something else for the roadmap :)

File: newsletter-june-2023.rst [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+Read the Docs newsletter - June 2023
+====================================
+
+News and updates
+----------------
+
+- ⚠️ A ``.readthedocs.yaml`` configuration file will be required for your future builds.
+  Read more about this change in :doc:`/migrate-configuration-v2`.
+- ✅️ Visiting ``/en/`` on a documentation project now redirects to ``/en/latest/``.
```
Comment (ericholscher) [inline newsletter-june-2023.rst:15]: This is true for any language, right? Should note that more explicitly.

File: newsletter-june-2023.rst [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+- 🍿️ All the talks from `Write the Docs Portland 2023 <https://www.writethedocs.org/conf/portland/2023/>`__ are ready to watch:
+  `Open the new playlist on Youtube <https://www.youtube.com/watch?v=EZJ0mk9Jj3s&list=PLZAeFn6dfHpneQPsDWa4OmEpgW4pNiaZ2>`__.
```
Comment (ericholscher) [inline newsletter-june-2023.rst:17]:
```suggestion
- 🍿️ All the talks from `Write the Docs Portland 2023 <https://www.writethedocs.org/conf/portland/2023/>`__, which we sponsored this year, are ready to watch:
  `Open the new playlist on Youtube <https://www.youtube.com/watch?v=EZJ0mk9Jj3s&list=PLZAeFn6dfHpneQPsDWa4OmEpgW4pNiaZ2>`__.
```

File: newsletter-june-2023.rst [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+- 🐛️ Builds with multiline commands suffered from `a few bugs <https://github.com/readthedocs/readthedocs.org/issues/10103>`__ that are hopefully all fixed now.
```
Comment (ericholscher) [inline newsletter-june-2023.rst:18]:
```suggestion
- 🐛️ Builds with multiline ``build.commands`` should now be `working reliably <https://github.com/readthedocs/readthedocs.org/issues/10103>`_.
```

File: newsletter-june-2023.rst [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+Possible issues
+---------------
+
+- ⚠️ Please make sure to read the blog post: :doc:`/migrate-configuration-v2`.
+
+  If you didn't have a ``.readthedocs.yaml`` configuration file,
+  and you are introducing one for the first time,
+  chances are that a lot of your documentation stack is outdated (Python, Sphinx, Ubuntu).
+  So your builds might break.
```
Comment (ericholscher) [inline newsletter-june-2023.rst:33]:
```suggestion
  and you are introducing one for the first time,
  we're excited for you to be able to use all our new features like specifying build tool versions (Node, Rust, etc.)!
```

**Review by ericholscher — APPROVED** [review]
Looks close, just a few small things to be more positive in our framing.

File: newsletter-june-2023.rst (hunk 3 above)
Comment (benjaoming) [reply newsletter-june-2023.rst]: True :+1: Fixed that one, it took a few more words to explain though :)

File: newsletter-june-2023.rst (hunk 6 above)
Comment (benjaoming) [reply newsletter-june-2023.rst]: Ah yes, that's more positive. Always hard to choose between the negative reasoning for taking action and the positive reasoning for making changes.

**Pull request merged**